pages/acknowledgments.tex

```diff
@@ -8,16 +8,16 @@
 
 
 
-\section*{About this rulebook}
-This is the official rulebook of the RoboCup@Home competition \YEAR.
+\section*{About This Rulebook}
+This is the official rulebook of the \YEAR ~RoboCup@Home competition.
 It has been written by the \YEAR ~RoboCup@Home Technical Committee with the special collaboration of (in alphabetical order):
 % Mauricio Matamoros,
 % and
 % Loy van Beek.
 
 
 
-\section*{How to cite this rulebook}
+\section*{How to Cite This Rulebook}
```

Comment:
`Cite` and `This` with capitals?

Reply:
AFAIK in English it is good practice to capitalize every word but pronouns in titles. *This* is a determiner, so according to some styles it gets capitalized.

Other conventions are more flexible and only require words of 4+ characters to be capitalized unless they are verbs, adjectives, pronouns, or nouns.

Maybe @justinhart can explain us further

Reply:
{How to Cite this Rulebook} would be the appropriate spelling.

Reply:
```suggestion
\section*{How to Cite this Rulebook}
```